### ISSUE TITLE
Setting a context-property (flow. or global.) reverts to msg. on save 

### ISSUE THREAD

**Issue by LorenzKahl** [mrgadget/node-red-contrib-eztimer#22]
eztimer: 1.1.6
node-red: 1.0.3

Tested in Chrome 79 & Firefox 59 on a Mac

When I'm trying to store a property in a context (flow. or global.) like so and save:
![image](https://user-images.githubusercontent.com/301135/72105409-cfbc7200-332d-11ea-9e98-4515ca4abb13.png)

...and then re-open the edit dialog the context is always reverted to msg.:
![image](https://user-images.githubusercontent.com/301135/72105437-d9de7080-332d-11ea-946a-24d8a4ecec8a.png)

I'm very new to node-red and don't know how to go about debugging this myself. 
Please let me know if I can be of any help. Maybe I'm just using this thing the wrong way ;-)

**Comment by mrgadget** [comment]
Hey, thanks for the detailed report. Welcome to node-red. We were all new once!

It’s not functionality I’ve used before so I’d say you’re right that it’s a bug. I’ll look further into this tonight and report back.

**Comment by mrgadget** [comment]
It would appear that I don't currently store/process the property type.  I'll look to get this rectified ASAP - as you've pointed out it looks like it should work, so it should work!

**Comment by LorenzKahl** [comment]
I didn't know there was a way of inspecting the context (at the time of opening the issue). It actually works fine and the context is written correctly. It's just that the UI doesn't reflect the setting if the node editor is opened again.

**Comment by mrgadget** [comment]
Hey @LorenzKahl, I don't think it was actually working - I didn't handle `flow` or `global` contexts at all in the node previously.  I _believe_ it's working now - I've committed the changes here - are you able to test (just need to replace the `index.js` & `index.html` files in the node's folder on node-red).

**Comment by LorenzKahl** [comment]
Hi @mrgadget, thanks!
I will have a look next weekend if that's ok with you.

**Comment by mrgadget** [comment]
Yep no stress, I’m guessing you’re the first that’s wanted to use this functionality so you’re only waiting on yourself now :) let me know how you get on. Cheers, Ben.

**Comment by LorenzKahl** [comment]
Hi Ben,

thanks for the changes. It fixes the un-persisted context dropdown. So I will close this issue.
I discovered another bug I think - the 'boolean' type is actually persisted as String "true" / "false".
I will open another issue ;-)

Cheers,
Lorenz
